templates/search.php

```diff
@@ -8,7 +8,9 @@
 						<header>
 							<h3 class="govuk-heading-m"><a href="<?php the_permalink() ?>"><?php the_title() ?></a></h3>
 							<?php
-		get_template_part('templates/entry-meta');
+		if (is_single()) {
```

Comment:
`is_single()` bases its result on the global `$wp_query`, see https://developer.wordpress.org/reference/functions/is_single/, so in the context of a search query is always going to return false. So currently, this hides the entry meta for all listings in a set of search results.

Instead, this probably wants to be something like `if(get_post_type() === 'post') {`